app/models/incomeSourceDetails/PropertyDetailsModel.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.incomeSourceDetails
+
+import java.time.LocalDate
+
+import models.core.{AccountingPeriodModel, CessationModel, ContactDetailsModel, CustomReads}
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{Json, Reads, _}
+
+case class PropertyDetailsModel(incomeSourceId: String,
+                                accountingPeriod: AccountingPeriodModel,
+                                contactDetails: Option[ContactDetailsModel],
+                                propertiesRented: Option[PropertiesRentedModel],
+                                cessation: Option[CessationModel],
+                                paperless: Option[Boolean],
+                                firstAccountingPeriodEndDate: Option[LocalDate])
```

Comment:
this could be reduced to incomeSourceId, accountingPeriod & firstAccountingPeriodEndDate

Reply:
Can it be refactored? I think this is part of of the response from DES as `IncomeSourceDetailsModel` depends on it.